content/en/apps/tutorials/user-management-tool.md

```diff
@@ -1,18 +1,26 @@
 ---
-title: "Configuring the CHT User Management tool"
+title: "CHT User Management tool"
 linkTitle: User management tool
 weight: 23
 description: >
- How to configure a web-based user management tool for your CHT project.
+ How to use and configure a user management tool for your CHT project.
 relatedContent: >
   apps/tutorials/contact-and-users-1
   apps/guides/data/users-bulk-load
 ---
 
-The user management tool is a user-friendly web application used within CHT projects to manage users. 
+The user management tool is a user-friendly web application used within CHT projects to manage users. It is used to decentralize the user management process while ensuring accuracy. The tool provides templates that can be downloaded and populated.
```

Comment:
The Community Health Toolkit (CHT) is highly configurable and it can be customized to support multiple hierarchies and users in the health care system. The CHT user management tool is a user friendly web application that works with CHT and is designed to support decentralization of the user management process to the subnational levels increasing efficiency and accuracy. This guide highlights the steps for setting up the user management tool. The guide has been customized for specific CHT supported national community health information systems and partners and ministries of health are advised to customize tool to fit their specific needs and contexts.